src/Symfony/Component/Console/Descriptor/JsonDescriptor.php

```diff
@@ -64,7 +64,7 @@ protected function describeCommand(Command $command, array $options = array())
     protected function describeApplication(Application $application, array $options = array())
     {
         $describedNamespace = isset($options['namespace']) ? $options['namespace'] : null;
```

Comment:
The suggestion of `getApplicationDescription` was to move option management to a single place. Keeping this doesnt solve that, gaining only 1 option * 4 descriptors.

Imo. we should rely on some `$description->getNamespace()` as of below, otherwise maybe revert.